apps/web-giddh/src/app/shared/eWayBill/eWayBill.store.ts

```
@@ -0,0 +1,33 @@
+import { Injectable, OnDestroy } from "@angular/core";
+import { ComponentStore } from "@ngrx/component-store";
+import { Store } from "@ngrx/store";
+import { AppState } from "../../store";
+import { Observable } from "rxjs";
+
+export interface EWayBillComponentState {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)